README.adoc

```diff
@@ -43,20 +43,20 @@ Learn how to check the health of microservices on Kubernetes by setting up readi
 
 == What you'll learn
 
-You will learn how to create a health check endpoint for your microservices. Then, you 
-will configure {kube} to use this endpoint to keep your microservices running smoothly.
+You will learn how to create health check endpoints for your microservices. Then, you 
+will configure {kube} to use these endpoints to keep your microservices running smoothly.
 
 MicroProfile Health allows services to report their health, and it publishes the overall 
-health status to a defined endpoint. If a service reports `UP`, then it's available. If the service reports `DOWN`, then it's unavailable.
-MicroProfile Health reports an individual service status at 
-the endpoint and indicates the overall status as `UP` if all the services are `UP`. A 
-service orchestrator can then use the health statuses to make decisions.
+health status to a defined endpoint. If a service reports `UP`, then it's available. If 
+the service reports `DOWN`, then it's unavailable. MicroProfile Health reports an individual 
+service status at the endpoint and indicates the overall status as `UP` if all the services 
+are `UP`. A service orchestrator can then use the health statuses to make decisions.
```

Comment:
Question about
"MicroProfile Health allows services to report their health, and it publishes the overall health status to a defined endpoint. If a service reports `UP`, then it's available. If the service reports `DOWN`, then it's unavailable. MicroProfile Health reports an individual service status at the endpoint and indicates the overall status as `UP` if all the services are `UP`."

Should "endpoint" be plural here? I know throughout the guide there's discussion of the `/health/live` and `/health/ready` endpoints, but does each service report its overall health to a single defined endpoint? That's how this reads to me.

Reply:
Yes I think it should be. Since `/health/live` will return the liveness health check report and `/health/ready` will return the readiness health check report. There is also `/health` which aggregates the data from `/live` and `/ready`.